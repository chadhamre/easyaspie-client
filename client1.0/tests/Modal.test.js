import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

<<<<<<< HEAD
import RestaurantModal from "../containers/ModalComponent.js";
=======
import RestaurantModal from '../containers/ModalComponent.js'
>>>>>>> search

const restaurantModal = renderer.create(<RestaurantModal />);
const RestaurantModalInstance = restaurantModal.root;
const rendered = restaurantModal.toJSON();
const restaurantModalShallow = shallow(<RestaurantModal />);
const restaurantModalRender = restaurantModalShallow.dive();

it("ModalComponent renders without crashing", () => {
  expect(rendered).toBeTruthy();
});

it("ModalComponent renders correctly", () => {
  expect(rendered).toMatchSnapshot();
});

it("ModalComponent should not be affected by props", () => {
  const restaurantModalShallow = shallow(<RestaurantModal foo="bar" />);
  expect(restaurantModalShallow).toMatchSnapshot();
  restaurantModalShallow.setProps({ random: "prop" });
  expect(restaurantModalShallow).toMatchSnapshot();
});

describe("Test the different render depending on the state", () => {
  it("Should not render if no state", () => {
    expect(restaurantModalRender.find("ScrollView").length).toBe(0);
  });
  // it('Should render once the state is set', () => {
  //   let restaurantModalShallow = shallow(
  //     <RestaurantModal />, {lifecycleExperimental: true}
  //   );
  //   restaurantModalShallow = restaurantModalShallow.setProps({isOpen: true})
  //   restaurantModalShallow.setState({restaurantInfo:{placeHolderData}});
  //   expect(restaurantModalShallow.dive().find('ScrollView')).toBe(1);
  // })
});

const placeHolderData = {
  address: "Not Available!",
  bestPhoto:
    "https://memegenerator.net/img/instances/500x/67453856/you-know-what-really-grinds-our-gears-too-muchtoo-little-information.jpg",
  categories: ["none"],
  counts: {},
  hours: ["never"],
  location: {
    lat: 0,
    lng: 0
  },
  name: "Unknown",
  names: {
    foursquare: "",
    google: ""
  },
  phone: "+66 66 66 66",
  photos: [
    {
      checkin: {
        createdAt: 0,
        id: "0",
        timeZoneOffset: 0,
        type: "none"
      },
      suffix: "/12013472_j0-aWZubfISESmExbsVFpBWX8GMUupXDghjVJj70Duc.jpg",
      user: {
        firstName: "Dan",
        gender: "male",
        id: "12013472",
        lastName: "Ciocoiu",
        photo: {
          prefix: "https://igx.4sqi.net/img/user/",
          suffix: "/ZQJNY12VTHWG4D1M.jpg"
        }
      },
      width: 720
    }
  ],
  place_id: "None",
  price: 0,
  prices: {
    foursquare: 9999
  },
<<<<<<< HEAD
  rating: 11,
  ratings: {
    google: 0
  }
};
=======
}
>>>>>>> search
