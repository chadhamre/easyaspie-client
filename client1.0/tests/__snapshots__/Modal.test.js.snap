// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalComponent renders correctly 1`] = `<View />`;

exports[`ModalComponent should not be affected by props 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RestaurantModal
    foo="bar"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "animationDuration": 400,
      "backButtonClose": false,
      "backdrop": true,
      "backdropColor": "black",
      "backdropContent": null,
      "backdropOpacity": 0.5,
      "backdropPressToClose": true,
      "children": <View
        style={
                Object {
                        "alignItems": "center",
                        "display": "flex",
                        "height": "100%",
                        "justifyContent": "center",
                        "position": "relative",
                        "width": "100%",
                      }
        }
>
        <withAnimatable(View)
                animation={
                        Object {
                                "0": Object {
                                  "bottom": 0,
                                  "opacity": 1,
                                },
                                "1": Object {
                                  "bottom": 10,
                                  "opacity": 0,
                                },
                              }
                }
                delay={0}
                direction="alternate"
                duration={undefined}
                easing={undefined}
                iterationCount="infinite"
                onAnimationBegin={[Function]}
                onAnimationEnd={[Function]}
                style={undefined}
                transition={undefined}
                useNativeDriver={false}
        >
                <Image
                        source={1}
                        style={
                                Object {
                                        "height": 29,
                                        "marginBottom": 6,
                                        "width": 30,
                                      }
                        }
                />
        </withAnimatable(View)>
        <Image
                source={1}
                style={
                        Object {
                                "height": 75,
                                "width": 150,
                              }
                }
        />
        <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                        Object {
                                "color": "#4eb9ce",
                                "fontFamily": "raleway",
                                "marginTop": 12,
                              }
                }
        >
                baking ...
        </Text>
</View>,
      "coverScreen": false,
      "easing": [Function],
      "keyboardTopOffset": 22,
      "onClosed": [Function],
      "onClosingState": undefined,
      "onOpened": [Function],
      "position": "center",
      "startOpen": false,
      "style": Object {
        "alignItems": "center",
        "flex": 1,
        "position": "absolute",
        "zIndex": 3,
      },
      "swipeArea": 250,
      "swipeThreshold": 50,
      "swipeToClose": true,
    },
    "ref": "modal1",
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <withAnimatable(View)
            animation={
                        Object {
                                    "0": Object {
                                      "bottom": 0,
                                      "opacity": 1,
                                    },
                                    "1": Object {
                                      "bottom": 10,
                                      "opacity": 0,
                                    },
                                  }
            }
            delay={0}
            direction="alternate"
            duration={undefined}
            easing={undefined}
            iterationCount="infinite"
            onAnimationBegin={[Function]}
            onAnimationEnd={[Function]}
            style={undefined}
            transition={undefined}
            useNativeDriver={false}
>
            <Image
                        source={1}
                        style={
                                    Object {
                                                "height": 29,
                                                "marginBottom": 6,
                                                "width": 30,
                                              }
                        }
            />
</withAnimatable(View)>,
          <Image
            source={1}
            style={
                        Object {
                                    "height": 75,
                                    "width": 150,
                                  }
            }
/>,
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
                        Object {
                                    "color": "#4eb9ce",
                                    "fontFamily": "raleway",
                                    "marginTop": 12,
                                  }
            }
>
            baking ...
</Text>,
        ],
        "style": Object {
          "alignItems": "center",
          "display": "flex",
          "height": "100%",
          "justifyContent": "center",
          "position": "relative",
          "width": "100%",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "animation": Object {
              "0": Object {
                "bottom": 0,
                "opacity": 1,
              },
              "1": Object {
                "bottom": 10,
                "opacity": 0,
              },
            },
            "children": <Image
              source={1}
              style={
                            Object {
                                          "height": 29,
                                          "marginBottom": 6,
                                          "width": 30,
                                        }
              }
/>,
            "delay": 0,
            "direction": "alternate",
            "duration": undefined,
            "easing": undefined,
            "iterationCount": "infinite",
            "onAnimationBegin": [Function],
            "onAnimationEnd": [Function],
            "style": undefined,
            "transition": undefined,
            "useNativeDriver": false,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "source": 1,
              "style": Object {
                "height": 29,
                "marginBottom": 6,
                "width": 30,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "source": 1,
            "style": Object {
              "height": 75,
              "width": 150,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "accessible": true,
            "allowFontScaling": true,
            "children": "baking ...",
            "disabled": false,
            "ellipsizeMode": "tail",
            "style": Object {
              "color": "#4eb9ce",
              "fontFamily": "raleway",
              "marginTop": 12,
            },
          },
          "ref": null,
          "rendered": "baking ...",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "animationDuration": 400,
        "backButtonClose": false,
        "backdrop": true,
        "backdropColor": "black",
        "backdropContent": null,
        "backdropOpacity": 0.5,
        "backdropPressToClose": true,
        "children": <View
          style={
                    Object {
                              "alignItems": "center",
                              "display": "flex",
                              "height": "100%",
                              "justifyContent": "center",
                              "position": "relative",
                              "width": "100%",
                            }
          }
>
          <withAnimatable(View)
                    animation={
                              Object {
                                        "0": Object {
                                          "bottom": 0,
                                          "opacity": 1,
                                        },
                                        "1": Object {
                                          "bottom": 10,
                                          "opacity": 0,
                                        },
                                      }
                    }
                    delay={0}
                    direction="alternate"
                    duration={undefined}
                    easing={undefined}
                    iterationCount="infinite"
                    onAnimationBegin={[Function]}
                    onAnimationEnd={[Function]}
                    style={undefined}
                    transition={undefined}
                    useNativeDriver={false}
          >
                    <Image
                              source={1}
                              style={
                                        Object {
                                                  "height": 29,
                                                  "marginBottom": 6,
                                                  "width": 30,
                                                }
                              }
                    />
          </withAnimatable(View)>
          <Image
                    source={1}
                    style={
                              Object {
                                        "height": 75,
                                        "width": 150,
                                      }
                    }
          />
          <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                              Object {
                                        "color": "#4eb9ce",
                                        "fontFamily": "raleway",
                                        "marginTop": 12,
                                      }
                    }
          >
                    baking ...
          </Text>
</View>,
        "coverScreen": false,
        "easing": [Function],
        "keyboardTopOffset": 22,
        "onClosed": [Function],
        "onClosingState": undefined,
        "onOpened": [Function],
        "position": "center",
        "startOpen": false,
        "style": Object {
          "alignItems": "center",
          "flex": 1,
          "position": "absolute",
          "zIndex": 3,
        },
        "swipeArea": 250,
        "swipeThreshold": 50,
        "swipeToClose": true,
      },
      "ref": "modal1",
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <withAnimatable(View)
              animation={
                            Object {
                                          "0": Object {
                                            "bottom": 0,
                                            "opacity": 1,
                                          },
                                          "1": Object {
                                            "bottom": 10,
                                            "opacity": 0,
                                          },
                                        }
              }
              delay={0}
              direction="alternate"
              duration={undefined}
              easing={undefined}
              iterationCount="infinite"
              onAnimationBegin={[Function]}
              onAnimationEnd={[Function]}
              style={undefined}
              transition={undefined}
              useNativeDriver={false}
>
              <Image
                            source={1}
                            style={
                                          Object {
                                                        "height": 29,
                                                        "marginBottom": 6,
                                                        "width": 30,
                                                      }
                            }
              />
</withAnimatable(View)>,
            <Image
              source={1}
              style={
                            Object {
                                          "height": 75,
                                          "width": 150,
                                        }
              }
/>,
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                            Object {
                                          "color": "#4eb9ce",
                                          "fontFamily": "raleway",
                                          "marginTop": 12,
                                        }
              }
>
              baking ...
</Text>,
          ],
          "style": Object {
            "alignItems": "center",
            "display": "flex",
            "height": "100%",
            "justifyContent": "center",
            "position": "relative",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "animation": Object {
                "0": Object {
                  "bottom": 0,
                  "opacity": 1,
                },
                "1": Object {
                  "bottom": 10,
                  "opacity": 0,
                },
              },
              "children": <Image
                source={1}
                style={
                                Object {
                                                "height": 29,
                                                "marginBottom": 6,
                                                "width": 30,
                                              }
                }
/>,
              "delay": 0,
              "direction": "alternate",
              "duration": undefined,
              "easing": undefined,
              "iterationCount": "infinite",
              "onAnimationBegin": [Function],
              "onAnimationEnd": [Function],
              "style": undefined,
              "transition": undefined,
              "useNativeDriver": false,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "source": 1,
                "style": Object {
                  "height": 29,
                  "marginBottom": 6,
                  "width": 30,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "source": 1,
              "style": Object {
                "height": 75,
                "width": 150,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "accessible": true,
              "allowFontScaling": true,
              "children": "baking ...",
              "disabled": false,
              "ellipsizeMode": "tail",
              "style": Object {
                "color": "#4eb9ce",
                "fontFamily": "raleway",
                "marginTop": 12,
              },
            },
            "ref": null,
            "rendered": "baking ...",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`ModalComponent should not be affected by props 2`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RestaurantModal
    foo="bar"
    random="prop"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "animationDuration": 400,
      "backButtonClose": false,
      "backdrop": true,
      "backdropColor": "black",
      "backdropContent": null,
      "backdropOpacity": 0.5,
      "backdropPressToClose": true,
      "children": <View
        style={
                Object {
                        "alignItems": "center",
                        "display": "flex",
                        "height": "100%",
                        "justifyContent": "center",
                        "position": "relative",
                        "width": "100%",
                      }
        }
>
        <withAnimatable(View)
                animation={
                        Object {
                                "0": Object {
                                  "bottom": 0,
                                  "opacity": 1,
                                },
                                "1": Object {
                                  "bottom": 10,
                                  "opacity": 0,
                                },
                              }
                }
                delay={0}
                direction="alternate"
                duration={undefined}
                easing={undefined}
                iterationCount="infinite"
                onAnimationBegin={[Function]}
                onAnimationEnd={[Function]}
                style={undefined}
                transition={undefined}
                useNativeDriver={false}
        >
                <Image
                        source={1}
                        style={
                                Object {
                                        "height": 29,
                                        "marginBottom": 6,
                                        "width": 30,
                                      }
                        }
                />
        </withAnimatable(View)>
        <Image
                source={1}
                style={
                        Object {
                                "height": 75,
                                "width": 150,
                              }
                }
        />
        <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                        Object {
                                "color": "#4eb9ce",
                                "fontFamily": "raleway",
                                "marginTop": 12,
                              }
                }
        >
                baking ...
        </Text>
</View>,
      "coverScreen": false,
      "easing": [Function],
      "keyboardTopOffset": 22,
      "onClosed": [Function],
      "onClosingState": undefined,
      "onOpened": [Function],
      "position": "center",
      "startOpen": false,
      "style": Object {
        "alignItems": "center",
        "flex": 1,
        "position": "absolute",
        "zIndex": 3,
      },
      "swipeArea": 250,
      "swipeThreshold": 50,
      "swipeToClose": true,
    },
    "ref": "modal1",
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <withAnimatable(View)
            animation={
                        Object {
                                    "0": Object {
                                      "bottom": 0,
                                      "opacity": 1,
                                    },
                                    "1": Object {
                                      "bottom": 10,
                                      "opacity": 0,
                                    },
                                  }
            }
            delay={0}
            direction="alternate"
            duration={undefined}
            easing={undefined}
            iterationCount="infinite"
            onAnimationBegin={[Function]}
            onAnimationEnd={[Function]}
            style={undefined}
            transition={undefined}
            useNativeDriver={false}
>
            <Image
                        source={1}
                        style={
                                    Object {
                                                "height": 29,
                                                "marginBottom": 6,
                                                "width": 30,
                                              }
                        }
            />
</withAnimatable(View)>,
          <Image
            source={1}
            style={
                        Object {
                                    "height": 75,
                                    "width": 150,
                                  }
            }
/>,
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
                        Object {
                                    "color": "#4eb9ce",
                                    "fontFamily": "raleway",
                                    "marginTop": 12,
                                  }
            }
>
            baking ...
</Text>,
        ],
        "style": Object {
          "alignItems": "center",
          "display": "flex",
          "height": "100%",
          "justifyContent": "center",
          "position": "relative",
          "width": "100%",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "animation": Object {
              "0": Object {
                "bottom": 0,
                "opacity": 1,
              },
              "1": Object {
                "bottom": 10,
                "opacity": 0,
              },
            },
            "children": <Image
              source={1}
              style={
                            Object {
                                          "height": 29,
                                          "marginBottom": 6,
                                          "width": 30,
                                        }
              }
/>,
            "delay": 0,
            "direction": "alternate",
            "duration": undefined,
            "easing": undefined,
            "iterationCount": "infinite",
            "onAnimationBegin": [Function],
            "onAnimationEnd": [Function],
            "style": undefined,
            "transition": undefined,
            "useNativeDriver": false,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "source": 1,
              "style": Object {
                "height": 29,
                "marginBottom": 6,
                "width": 30,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "source": 1,
            "style": Object {
              "height": 75,
              "width": 150,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "accessible": true,
            "allowFontScaling": true,
            "children": "baking ...",
            "disabled": false,
            "ellipsizeMode": "tail",
            "style": Object {
              "color": "#4eb9ce",
              "fontFamily": "raleway",
              "marginTop": 12,
            },
          },
          "ref": null,
          "rendered": "baking ...",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "animationDuration": 400,
        "backButtonClose": false,
        "backdrop": true,
        "backdropColor": "black",
        "backdropContent": null,
        "backdropOpacity": 0.5,
        "backdropPressToClose": true,
        "children": <View
          style={
                    Object {
                              "alignItems": "center",
                              "display": "flex",
                              "height": "100%",
                              "justifyContent": "center",
                              "position": "relative",
                              "width": "100%",
                            }
          }
>
          <withAnimatable(View)
                    animation={
                              Object {
                                        "0": Object {
                                          "bottom": 0,
                                          "opacity": 1,
                                        },
                                        "1": Object {
                                          "bottom": 10,
                                          "opacity": 0,
                                        },
                                      }
                    }
                    delay={0}
                    direction="alternate"
                    duration={undefined}
                    easing={undefined}
                    iterationCount="infinite"
                    onAnimationBegin={[Function]}
                    onAnimationEnd={[Function]}
                    style={undefined}
                    transition={undefined}
                    useNativeDriver={false}
          >
                    <Image
                              source={1}
                              style={
                                        Object {
                                                  "height": 29,
                                                  "marginBottom": 6,
                                                  "width": 30,
                                                }
                              }
                    />
          </withAnimatable(View)>
          <Image
                    source={1}
                    style={
                              Object {
                                        "height": 75,
                                        "width": 150,
                                      }
                    }
          />
          <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                              Object {
                                        "color": "#4eb9ce",
                                        "fontFamily": "raleway",
                                        "marginTop": 12,
                                      }
                    }
          >
                    baking ...
          </Text>
</View>,
        "coverScreen": false,
        "easing": [Function],
        "keyboardTopOffset": 22,
        "onClosed": [Function],
        "onClosingState": undefined,
        "onOpened": [Function],
        "position": "center",
        "startOpen": false,
        "style": Object {
          "alignItems": "center",
          "flex": 1,
          "position": "absolute",
          "zIndex": 3,
        },
        "swipeArea": 250,
        "swipeThreshold": 50,
        "swipeToClose": true,
      },
      "ref": "modal1",
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <withAnimatable(View)
              animation={
                            Object {
                                          "0": Object {
                                            "bottom": 0,
                                            "opacity": 1,
                                          },
                                          "1": Object {
                                            "bottom": 10,
                                            "opacity": 0,
                                          },
                                        }
              }
              delay={0}
              direction="alternate"
              duration={undefined}
              easing={undefined}
              iterationCount="infinite"
              onAnimationBegin={[Function]}
              onAnimationEnd={[Function]}
              style={undefined}
              transition={undefined}
              useNativeDriver={false}
>
              <Image
                            source={1}
                            style={
                                          Object {
                                                        "height": 29,
                                                        "marginBottom": 6,
                                                        "width": 30,
                                                      }
                            }
              />
</withAnimatable(View)>,
            <Image
              source={1}
              style={
                            Object {
                                          "height": 75,
                                          "width": 150,
                                        }
              }
/>,
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                            Object {
                                          "color": "#4eb9ce",
                                          "fontFamily": "raleway",
                                          "marginTop": 12,
                                        }
              }
>
              baking ...
</Text>,
          ],
          "style": Object {
            "alignItems": "center",
            "display": "flex",
            "height": "100%",
            "justifyContent": "center",
            "position": "relative",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "animation": Object {
                "0": Object {
                  "bottom": 0,
                  "opacity": 1,
                },
                "1": Object {
                  "bottom": 10,
                  "opacity": 0,
                },
              },
              "children": <Image
                source={1}
                style={
                                Object {
                                                "height": 29,
                                                "marginBottom": 6,
                                                "width": 30,
                                              }
                }
/>,
              "delay": 0,
              "direction": "alternate",
              "duration": undefined,
              "easing": undefined,
              "iterationCount": "infinite",
              "onAnimationBegin": [Function],
              "onAnimationEnd": [Function],
              "style": undefined,
              "transition": undefined,
              "useNativeDriver": false,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "source": 1,
                "style": Object {
                  "height": 29,
                  "marginBottom": 6,
                  "width": 30,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "source": 1,
              "style": Object {
                "height": 75,
                "width": 150,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "accessible": true,
              "allowFontScaling": true,
              "children": "baking ...",
              "disabled": false,
              "ellipsizeMode": "tail",
              "style": Object {
                "color": "#4eb9ce",
                "fontFamily": "raleway",
                "marginTop": 12,
              },
            },
            "ref": null,
            "rendered": "baking ...",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
